tfgcv/validate_assets.go

```diff
@@ -32,11 +32,19 @@ func BuildVersion() string {
 	return buildVersion
 }
 
-// ValidateAssets instantiates GCV and audits CAI assets.
+// ValidateAssets instantiates GCV and audits CAI assets using "policies"
+// and "lib" folder under policyPath.
 func ValidateAssets(assets []google.Asset, policyPath string) (*validator.AuditResponse, error) {
```

Comment:
Could we give the two policyPath arguments different names since they represent different values now? Maybe change this policyPath to `policyRootPath`?

Reply:
Done.